package.json

```diff
@@ -68,6 +68,13 @@
     "url-loader": "^0.5.7",
     "webpack": "^4.16.2",
     "webpack-bundle-tracker": "0.0.93",
-    "webpack-livereload-plugin": "^2.1.1"
+    "webpack-livereload-plugin": "^2.1.1",
+    "autoprefixer": "^9.8.4",
+    "cssnano": "^4.1.10",
+    "gulp": "^4.0.2",
+    "gulp-postcss": "^8.0.0",
+    "gulp-sass": "^4.1.0",
+    "gulp-sourcemaps": "^2.6.5",
+    "node-sass": "^5.0.0"
```

Comment:
Can we not add any of these? We already have a build system (webpack). Why add gulp?

Especially `node-sass`, which requires C++ code compilation and does not reliably build on Windows. https://github.com/OpenDroneMap/WebODM/commit/85b2a8a0db688f30ebbec1fd87b14d25164f5d50

Reply:
If you don't want to setup the build system (simpler), build the static CSS files locally however you want and commit them to the repo. The admin area doesn't get edited very often (it's a built-in component), so it's OK to do that.

Reply:
Removed gulp in https://github.com/OpenDroneMap/WebODM/pull/1129/commits/a325cf1f9858c7606254726197f1f65fbeab81b3, there is a tool in vscode that does it automatically https://marketplace.visualstudio.com/items?itemName=ritwickdey.live-sass

Reply:
You might have to commit the CSS files back into the repo though.

Reply:
I think they are in the repo already https://github.com/HeDo88TH/WebODM/tree/improve-responsiveness/app/static/admin/css